### ISSUE TITLE
dynamic_orientation still works even though it's lock at device

### ISSUE THREAD

**Issue by Dragosha** [defold/editor2-issues#1438]
If we turn on "dynamic_orientation" in  https://www.defold.com/manuals/project-settings/#_display the app will rotate screen orientation even though it's lock at device properties.

It looks like a engine bug and does reproduced on any Android device.

**Comment by subsoap** [comment]
This is known tohappen on debug but shouldn't happen in release afaik.

**Comment by Dragosha** [comment]
> This is known to happen on debug but shouldn't happen in release afaik.
but it's happened in release bundle. I've tons of reports from players of my game in Google.Play after last update where I've turn on this key. And it's release .apk.

**Comment by subsoap** [comment]
Are you using a custom Android manifest file? You might need to get a new copy from a generated version and replace with custom one if something changed related to that. Otherwise you should be able to hard force one orientation with a manifest setting.

**Comment by Dragosha** [comment]
Yes, I used custom manifest file. Thanks, I'll check difference between my file and generated.

**Comment by Dragosha** [comment]
looks like no defference there.
My file contains `android:screenOrientation="{{orientation-support}}"` string, so in bundle it change to 
`android:screenOrientation="sensor"` as in default generated manifest.

**Comment by andsveking** [comment]
Added a engine ticket; DEF-3010

**Comment by AGulev** [comment]
For now static orientation works only in one side (for landscape it's only left - it use android:screenOrientation = "landscape")
Would be great to have  an option to setup android:screenOrientation = "sensorLandscape" for using left and right landscape.
